site/management.md

```diff
@@ -335,6 +348,61 @@ curl -i --header "authorization: Bearer &lt;token&gt;" http://localhost:15672/ap
 This is true for all endpoints except `GET /definitions` and `POST /definitions`. Those
 endpoints require the token to be passed in the `token` query string parameter.
 
+### Configure Scopes to request during authentication
+
+We can configure which OAuth2 **scopes** RabbitMQ should claim when redirecting the user to the authorization server by
```

Comment:
"We can configure...."

Should that be "You can configure"

This is a GLOBAL comment wherever there is reference to "we" if it makes sense to change it to "you" I presume it will be the user/customer doing this?